data/136/069/852/7/1360698527.geojson

```diff
@@ -261,7 +261,7 @@
         "gla",
         "gle"
     ],
-    "wof:lastmodified":1694493157,
+    "wof:lastmodified":1695884423,
```

Comment:
Updated, but no changes here?

Reply:
Workflow wise, I should exclude from the CSVs any records I don't think should be modified, or only exportify the record if there was a change (so new logic to track if there was a change).

The affects ~ 1632 records, almost all of which are localadmin, in a few dozen countries (but just a handful of countries for coarse admin placetypes).

I'll look into unsetting the changes for just those records...

Reply:
We discussed in Slack and OK to skip as it's a 0.1% problem across all the related changes on 169k records that doesn't have negative consequences.